### PR TITLE
Cached bundler

### PR DESCRIPTION
**Pull request by thedrow**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 language: ruby
+sudo: false
```
Comment (michaelklishin) [inline .travis.yml]: This is wrong. `sudo` is used because `rabbitmqctl` is used by `bin/ci/before_script`.

**Comment by michaelklishin** [comment]
Please consider investigating how the build system works before you submit pull requests. `sudo` is required and https://github.com/ruby-amqp/amq-protocol/pull/60 also introduced a change that likely was never tested.

**Comment by thedrow** [comment]
Are you sure that rabbitmqctl always requires sudo? I think that rabbitmq is running under the travis user.

**Comment by michaelklishin** [comment]
Not always but it does on Travis. RabbitMQ is provisioned using a Debian package there:
https://github.com/ruby-amqp/bunny/blob/master/bin/ci/before_build

**Comment by michaelklishin** [comment]
Yeah, a quick glance at the build failure for this PR answers the above question.

**Pull request closed without merging**